sphinx/builders/latex/__init__.py

```diff
@@ -126,11 +128,14 @@ class LaTeXBuilder(Builder):
 
     def init(self):
         # type: () -> None
+        self.context = {}           # type: Dict[unicode, Any]
```

Comment:
Note: In this PR, the responsibility of context object is unbalanced.

* Builder: creating context object
* Writer: updating context object and generate .tex file using context

I think it should be refactored this in future. It would be better we should move the responsibility of generating .tex file to builder module (like HTML builder does).